src/routing/mod.rs

```diff
@@ -192,6 +192,15 @@ impl Routing {
         self.dispatcher.clone().handle_commands(command).await
     }
 
+    /// Signals the Elders of our section to test connectivity to a node.
+    pub async fn start_connectivity_test(&self, name: XorName) -> Result<()> {
+        if !self.is_elder().await {
```

Comment:
thought it's said that Adult shall notify Elder to start a connectivity test?

Reply:
Ah yes that's right. I'll remove this condition :+1: